### ISSUE TITLE
Nucleus proxy not functioning

### ISSUE THREAD

**Issue by jxckUK** [TGRHavoc/live_map#83]
**Describe the bug**

**To Reproduce**
1. Enable nucleus in the server configuration
2. Collect the WSS link from the console output
3. Attempt to connect to the WSS link from any environment

**Expected behavior**
This should be the simplest way to get access to the live map resource via the automatic reverse proxy setup.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - Windows 11
 - Edge
 - N/A

**Additional context/Screenshots**
N/A

**Comment by rhys19** [comment]
> **Describe the bug** A clear and concise description of what the bug is.
> 
> **To Reproduce** Steps to reproduce the behavior:
> 
>     1. Enable nucleus in the server configuration
> 
>     2. Collect the WSS link from the console output
> 
>     3. Attempt to connect to the WSS link from any environment
> 
>     4. See error
> 
> 
> **Expected behavior** This should be the simplest way to get access to the live map resource via the automatic reverse proxy setup.
> 
> **Screenshots** N/A
> 
> **Desktop (please complete the following information):**
> 
>     * Windows 11
> 
>     * Edge
> 
>     * N/A
> 
> 
> **Additional context/Screenshots** N/A

Technically you missed the description of it so I'll do it for you:
Nucleas proxy will not work, when you use the nucleas link it provides you it says can't connect to websocket.

You'll probably change up this description because idk what the real error is been awhile since I touched it.

but for Havoc this issue isn't just him it's me too, I can't seem to use nucleas either, and for some reason using a reverse proxy via NGINX doesn't anymore either when I had it setup on your old one before the update for nucleas and it worked flawlessly.

**Comment by jxckUK** [comment]
My bad, thanks for adding. I'm also aware Havok knows about this issue (he's stated that he is on CFC forums) hence I didn't go into too much detail.

**Comment by TGRHavoc** [comment]
> this issue isn't just him it's me too, I can't seem to use nucleas either, and for some reason using a reverse proxy via NGINX doesn't anymore either when I had it setup on your old one before the update for nucleas and it worked flawlessly.

Yeah, as @jxckUK mentioned I already know about Nucelus not working. It's because of my hacky implementation to set up a reverse proxy using Nucleus... Don't have much spare time to develop and test a fix at this moment in time. Afaik, I might not even need to touch the Nucleus API but, again. Need time to test this theory 😞

**Comment by jxckUK** [comment]
> > this issue isn't just him it's me too, I can't seem to use nucleas either, and for some reason using a reverse proxy via NGINX doesn't anymore either when I had it setup on your old one before the update for nucleas and it worked flawlessly.
> 
> Yeah, as @jxckUK mentioned I already know about Nucelus not working. It's because of my hacky implementation to set up a reverse proxy using Nucleus... Don't have much spare time to develop and test a fix at this moment in time. Afaik, I might not even need to touch the Nucleus API but, again. Need time to test this theory 😞

No worries! - Like Rhys, I have tried using the standalone method and not had much luck either 😞

**Comment by ViolentMinds** [comment]
I wish this worked :(

**Comment by TGRHavoc** [comment]
> I wish this worked :(

Will be fixed when v4 is out with SSE. You will be able to use the default Nucleus url that's generated for the server since it'll just hook the SetHttpHandler native.
Going to mark as "wont fix" as technically I'm not touching Websockets with the newer version.

**Comment by Legacy-TacticalGamingInteractive** [comment]
sounds good
